src/main.rs

```diff
@@ -16,14 +17,18 @@ pub struct ExplorerCli {
     #[command(subcommand)]
     command: Command,
 
-    /// The url/endpoint to use for any rpc requests.
-    #[arg(
-        long,
-        short = 'u',
-        default_value = "http://api.mainnet-beta.solana.com",
-        global = true
-    )]
-    rpc_url: String,
+    /// Optionally include your RPC endpoint. Use "local", "dev", "main" for default endpoints. Defaults to your Solana CLI config file.
```

Comment:
Should also suggest shorthand versions, e.g. `-ul`.

Reply:
added alias command for `rpc_url` arg

https://github.com/cavemanloverboy/sol/blob/d7e388ab7964612b087b33bfba72705e7c98eece/src/main.rs#L21